packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js

```diff
@@ -534,7 +542,9 @@ export function JSONLike({ onSubmit, className, style, ...props }) {
 	if (rootIsObject) {
 		if (props.rootLabel) {
 			labelId = (props.id && `${props.id}-label`) || 'tc-object-viewer-label';
-			label = (
+			label = props.hideTooltip ? (
+				<div className={theme['root-label-overflow']}>{props.rootLabel}</div>
```

Comment:
[Optional]
used twice so may be we can have a const,
const labelText = <div className={theme['root-label-overflow']}>{props.rootLabel}</div>;